### PR TITLE
 Adding my profile to the the public/data/

### PR DESCRIPTION
**Pull request by imran1509**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1830"></a>

### CODE REVIEW

**Review by imran1509 — COMMENTED** [review]
Looks Okay to me

File: public/data/imran1509.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  "bio": "I am DevOps Engineer from India. I work with tools like Kubernetes, Docker, Git & GitHub, Ansible, Terraform, Jenkins, AWS, Linux-ubuntu etc. Python & Golang.",
+  "avatar": "https://github.com/imran1509.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/imran1509",
+      "icon": "github"
+    },
+    {
+      "name": "Connect with me on Linkedin",
```
Comment (Panquesito7) [inline public/data/imran1509.json:13]: I recommend capitalizing the `i` here. What do you think?
```suggestion
      "name": "Connect with me on LinkedIn",
```

File: public/data/imran1509.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "url": "https://www.linkedin.com/in/mohdimran1995/",
+      "icon": "linkedin"
+    },
+    {
+      "name": "Follow me on twitter",
```
Comment (Panquesito7) [inline public/data/imran1509.json:18]: I recommend capitalizing the `t` here. What do you think?
```suggestion
      "name": "Follow me on Twitter",
```

**Review by Panquesito7 — APPROVED** [review]
Looks great! Thanks for contributing. 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 🚀🚀

**Pull request merged**